src/app.js

```diff
@@ -2,0 +1,28 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const path = require('path');
+
+const fun = (firsts, seconds) => {
+  if (!fs.existsSync(firsts)) {
+    console.error(`Source file does not exist: ${firsts}`);
+
+    return;
+  }
+
+  const destination =
+    fs.existsSync(seconds) && fs.statSync(seconds).isDirectory()
+      ? path.join(seconds, path.basename(firsts))
+      : seconds;
+
+  try {
+    fs.renameSync(firsts, destination);
```

Comment:
The `fs.renameSync` method is used here to move the file. This will throw an error if the destination file already exists. Consider handling this case by either overwriting the file or notifying the user.